rawpheno.install

```diff
@@ -499,14 +522,17 @@ function rawpheno_install() {
 
     // R version. To be implemented to all column headers.
     $r_version = rawpheno_function_make_r_compatible($header);
-    chado_insert_record('cvtermprop',
-      array(
-        'cvterm_id' => $cvterm->cvterm_id,
-        'type_id'   => $arr_cvid_cvname[ $cv['cv_rver'] ],
-        'value'     => $r_version,
-        'rank'      => 0
-      )
-    );
+
+    if ($cvterm) {
+      chado_insert_record('cvtermprop',
+        array(
+          'cvterm_id' => $cvterm->cvterm_id,
+          'type_id'   => $arr_cvid_cvname[ $cv['cv_rver'] ],
+          'value'     => $r_version,
+          'rank'      => 0
+        )
+      );
+    }
```

Comment:
Everytime you use an if like this, you should have an else with a tripal_report_error(). Otherwise, you may be masking the problem.